### PR TITLE
a4term: init at 0.2.2

### PR DESCRIPTION
**Pull request by onemoresuza**
###### Description of changes

[Package request](https://github.com/NixOS/nixpkgs/issues/218328).

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/a4term/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libtickit
+, libvterm-neovim
+}:
+stdenv.mkDerivation rec {
```
Comment (AndersonTorres) [inline pkgs/tools/misc/a4term/default.nix:7]:
```suggestion

stdenv.mkDerivation rec {
```

File: pkgs/tools/misc/a4term/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "rpmohn";
+    repo = "a4";
+    rev = "v${version}";
+    hash = "sha256-hsAEiPOZBqjvmSZEmZwfDqHZV/8ym62RZPxl3DG4ntQ=";
+  };
+
+  buildInputs = [ libtickit libvterm-neovim ];
```
Comment (AndersonTorres) [inline pkgs/tools/misc/a4term/default.nix:18]:
```suggestion
  buildInputs = [
    libtickit
    libvterm-neovim 
  ];
```

File: pkgs/tools/misc/a4term/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with lib; {
```
Comment (AndersonTorres) [inline pkgs/tools/misc/a4term/default.nix:22]:
```suggestion
  meta = {
```

File: pkgs/tools/misc/a4term/default.nix [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    description = "A dynamic terminal window manager";
+    homepage = "https://www.a4term.com/";
+    license = licenses.mit;
```
Comment (AndersonTorres) [inline pkgs/tools/misc/a4term/default.nix:25]:
```suggestion
    license = lib.licenses.mit;
```

File: pkgs/tools/misc/a4term/default.nix [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    maintainers = with maintainers; [ onemoresuza ];
```
Comment (AndersonTorres) [inline pkgs/tools/misc/a4term/default.nix:26]:
```suggestion
    maintainers = with lib.maintainers; [ onemoresuza ];
```

File: pkgs/tools/misc/a4term/default.nix [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    platforms = platforms.linux;
```
Comment (AndersonTorres) [inline pkgs/tools/misc/a4term/default.nix:27]:
```suggestion
    platforms = lib.platforms.linux;
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Comment by onemoresuza** [comment]
Requested changes made, @AndersonTorres.

**Review by AndersonTorres — APPROVED** [review]
ping me tomorrow to merge

**Pull request merged**